engine/exchange_manager.go

```diff
@@ -201,7 +212,65 @@ func (m *ExchangeManager) NewExchangeByName(name string) (exchange.IBotExchange,
 		if m.Builder != nil {
 			return m.Builder.NewExchangeByName(nameLower)
 		}
-		return nil, fmt.Errorf("%s, %w", nameLower, ErrExchangeNotFound)
+		return nil, fmt.Errorf("exchange manager: %s, %w", nameLower, ErrExchangeNotFound)
 	}
 	return exch, nil
 }
+
+// Shutdown shuts down all exchanges and unloads them
+func (m *ExchangeManager) Shutdown() error {
+	if m == nil {
+		return fmt.Errorf("exchange manager: %w", ErrNilSubsystem)
+	}
+
+	var mtx sync.Mutex
+	timer := time.NewTimer(time.Second * 10)
```

Comment:
```suggestion
	timer := time.NewTimer(time.Second * 10)
```

Would be nice to see this configurable instead of a magic number